python/taichi/lang/kernel_impl.py

```diff
@@ -655,10 +655,15 @@ def func__(*args):
 
             actual_argument_slot = 0
             launch_ctx = t_kernel.make_launch_context()
```

Comment:
I think this check should be done within launch context...

Reply:
In fact this should be done when compiling the kernel. Let's keep this as a temporary solution. I'll refactor it later.

Reply:
BTW the limit will be removed as soon as the refactoring completes.